site_src/Assets/input.css

```
@@ -0,0 +1,329 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+body {
+  font-family: 'Outfit', ui-sans-serif, system-ui, -apple-system, sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)